xml-parser/src/test/java/org/uniprot/core/xml/uniprot/comment/SubcellLocationNameMapImplTest.java

```diff
@@ -0,0 +1,27 @@
+package org.uniprot.core.xml.uniprot.comment;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class SubcellLocationNameMapImplTest {
+
+    private static final String LOCATION_NAME = "Plastid";
+    private static final String SUBCELL_FILE = "subcell.txt";
+
+    @Test
+    void canGetLocationName() {
+        SubcellLocationNameMapImpl subcellLocationNameMap =
+                new SubcellLocationNameMapImpl(SUBCELL_FILE);
+        assertNotNull(subcellLocationNameMap);
```

Comment:
probably not needed, but it might be possible to ensure and subsequent calls do not make the full text file be parsed each time ... just a comment ...